Firestore/Source/Remote/FSTWatchChange.mm

```diff
@@ -109,11 +114,11 @@ - (BOOL)isEqual:(id)other {
   }
 
   FSTExistenceFilterWatchChange *otherChange = (FSTExistenceFilterWatchChange *)other;
-  return [_filter isEqual:otherChange->_filter] && _targetID == otherChange->_targetID;
+  return _filter == otherChange->_filter && _targetID == otherChange->_targetID;
 }
 
 - (NSUInteger)hash {
-  return self.filter.hash;
+  return _filter.count();
```

Comment:
This used to be the implementation of `FSTExistenceFilter hash`. Inlining it here allows keeping `ExistenceFilter` a pure C++ class.